src/website/views/components/sidenav/_examples.pug

```diff
@@ -510,7 +510,9 @@ h3 Open on hover
                   if val2 === 'A' || val2 === 'B'
                     li(class=`${val1 === active.menuItem && val2 === active.drawerMenuItem ? '-active -expanded' : ''}`)
                       a(href=`#drawer-item-list-${val1}-${val2}`)
-                        .chi-sidenav__title=`Title ${String.fromCharCode(val2.charCodeAt(0) + (drawerMenuItems.length * (val1 - 1)))}`
+                        .chi-sidenav__title.-position--relative
+                          span=`Title ${String.fromCharCode(val2.charCodeAt(0) + (drawerMenuItems.length * (val1 - 1)))}`
+                          i.chi-icon.icon-chevron-down.-xs.chi-sidenav__drawer-item-chevron
```

Comment:
Can we render the icons dynamically as a background image on drawer list items that contain submenus?

This is highlighting another another minor issue, no padding-right has been defined on the list item `a` tag. There is a 1.5rem padding-left defined on the `a` tag so the chevron should render with a right: 1.5rem to ensure equal spacing on both sides. This will translate to a padding-right: 1.5rem for list items with no submenus/chevron icons and a padding-right: 3rem for items that do.